web/src/views/worker/components/WorkerForm.vue

```diff
@@ -0,0 +1,50 @@
+<!--
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+-->
+
+<script lang="ts" setup>
+const emits = defineEmits(['search', 'reset'])
+
+const formData = reactive({
+  master: ''
+})
+
+const search = () => {
+  emits('search', unref(formData))
+}
+
+const reset = () => {
+  formData.master = ''
+  emits('reset', unref(formData))
+}
+</script>
+
+<template>
+  <n-form :model="formData" :show-feedback="false" label-placement="left" @keydown.enter="search">
+    <n-grid :x-gap="24" :y-gap="24" :cols="4">
+      <n-form-item-gi label="Master" path="master">
```

Comment:
Could we support search with other fields like `HostName` etc?

Reply:
which fields specifically need to be searched?